scheduler.rst

```diff
@@ -286,14 +286,37 @@ defined by PHP datetime functions::
     RecurringMessage::every('3 weeks', new Message());
     RecurringMessage::every('first Monday of next month', new Message());
 
-    $from = new \DateTimeImmutable('13:47', new \DateTimeZone('Europe/Paris'));
-    $until = '2023-06-12';
-    RecurringMessage::every('first Monday of next month', new Message(), $from, $until);
-
 .. tip::
 
     You can also define periodic tasks using :ref:`the AsPeriodicTask attribute <scheduler-attributes-periodic-task>`.
 
+Be aware that the message isn't passed to the messenger when you start the
+scheduler. The message will only be executed after the first frequency period
+has passed.
```

Comment:
```suggestion
The message will be executed after the first frequency period
has passed.
```

Documentation are often easier to read when written in positive / direct tone. Here i think we should remove "be aware" or "only", "isnt". 

(suggestion just for illustration)

Reply:
I agree. We reworded this a bit while merging. Thanks.